onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb

```diff
@@ -916,34 +931,27 @@
     "| Engine | Full Precision(MB) | Quantized(MB) |\n",
     "| --- | --- | --- |\n",
     "| PyTorch 1.6 | 417.7 | 173.1 |\n",
-    "| ORT 1.4 | 417.7 | 106.5 |\n",
+    "| ORT 1.5 | 417.7 | 106.5 |\n",
     "\n",
     "### Accuracy\n",
     "OnnxRuntime achieves a little bit better accuracy and F1 score, even though it has small model size.\n",
     "\n",
-    "| Metrics | Full Precision | PyTorch 1.6 Quantization | ORT 1.4 Quantization |\n",
+    "| Metrics | Full Precision | PyTorch 1.6 Quantization | ORT 1.5 Quantization |\n",
```

Comment:
```suggestion
    "| Metrics | Full Precision | PyTorch 1.6 Quantization | ORT 1.5.1 Quantization |\n",
```

Reply:
Usually, patch version is not specified in benchmark